docs/upgrades.md

```diff
@@ -0,0 +1,196 @@
+
+You will always want to upgrade the server before upgrading the clients. If the
+server is being upgraded to a version that still supports the version being
+used by your clients, you can often upgrade the server without any downtime for
+users. This is one benefit of keeping up-to-date with regular upgrades! If you
+want to know what client versions a given server supports, you can look at
+the [release notes](https://github.com/sematic-ai/sematic/releases).
+
+Often, you will need to upgrade the server beyond support for the clients
+currently in use. When this happens, a recommended upgrade workflow is:
+
+1. Draft a PR in your repo that upgrades the clients
+2. Notify users that there will be an upcoming maintenance window for upgrades
+   during which they may not be able to use Sematic.
+3. When the maintenance window arrives, upgrade the server.
+4. Test the clients using the branch for your upgrade PR. Assuming all goes
+   well, merge the PR. If not, roll back the server.
+5. Notify users that the maintenance window is now complete and that they will
+   need to merge the main branch into their feature branches to begin using the
+   latest version.
+
+## From vX1.X2.X3 to vY1.Y2.Y3
+
+This section contains instructions for migrating from one
+version of Sematic to another. It does not cover every version
+delta, but only (a) from one version to a subsequent one (b)
+upgrades where there are non-trivial details to pay attention to
+when performing the upgrade.
+
+### v0.23.X to v0.24.Y
+
+#### Server/Helm upgrade notes
+
+You will need to make the following changes to the `values.yaml`
+for your Helm deployment:
+
+- If you are NOT using Sematic "Enterprise Edition",
+change `image.repository` from `sematicai/sematic-server`
+to `sematic/sematic-server`.
+- You may have hard-coded a Sematic version in `image.tag`
+to work around the bug referenced in
+[this PR](https://github.com/sematic-ai/sematic/pull/534). If so,
+you may now remove the hard code of `image.tag` to again rely
+on the default behavior of using the version from the
+[`Chart.yaml`](https://github.com/sematic-ai/sematic/blob/main/helm/sematic-server/Chart.yaml)
+for the chart you install.
+- If you are specifying `ingress.dashboard_url`, change the name of this
+parameter to
+[`ingress.sematic_dashboard_url`](https://github.com/sematic-ai/sematic/pull/547)
+
+## FOSS to "Enterprise Edition"[^1]
+
+(0) Reach out to Sematic to obtain a license for "Sematic EE."
+(1) In your helm deployment, change `image.repository` to `sematic/sematic-server-ee`.
+(2) In your `pip` installation (client installation), change from depending on `sematic`
+to `sematic[<extra>]`, where `<extra>` is the appropriate variant of Sematic containing
+the EE features you are interested in. See [integrations](#integrations) below for
+specific extras. Note that `sematic[all]` will equip clients with ALL Enterprise Edition
+features.
+
+## Integrations
```

Comment:
Could you please extract this to a separate EE integrations page, and have this page deal only with upgrades?